mergify_engine/duplicate_pull.py

```diff
@@ -307,19 +308,33 @@ async def duplicate(
     finally:
         await git.cleanup()
 
-    body = (
-        f"This is an automatic {kind} of pull request #{ctxt.pull['number']} done by [Mergify](https://mergify.io)."
-        + body
-    )
-
-    if cherry_pick_fail:
-        body += (
-            "To fixup this pull request, you can check out it locally. "
+    if cherry_pick_error:
+        cherry_pick_error += (
+            "To fix up this pull request, you can check it out locally. "
             "See documentation: "
             "https://help.github.com/articles/"
             "checking-out-pull-requests-locally/"
         )
 
+    try:
+        title = await ctxt.pull_request.render_template(
+            title_template,
+            extra_variables={"destination_branch": branch_name},
+        )
+    except context.RenderTemplateFailure as rmf:
+        raise DuplicateFailed(f"Invalid title message: {rmf}")
+
+    try:
+        body = await ctxt.pull_request.render_template(
+            body_template,
+            extra_variables={
+                "destination_branch": branch_name,
+                "cherry_pick_error": cherry_pick_error or "",
```

Comment:
```suggestion
                "cherry_pick_error": cherry_pick_error,
```
I don't think it can be anything else than a `str`